integration/store_gateway_limits_hit_test.go

```diff
@@ -0,0 +1,145 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+//go:build requires_docker
+// +build requires_docker
+
+package integration
+
+import (
+	"net/http"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/grafana/e2e"
+	e2edb "github.com/grafana/e2e/db"
+	"github.com/prometheus/prometheus/prompb"
+	"github.com/stretchr/testify/require"
+
+	"github.com/grafana/mimir/integration/e2emimir"
+	"github.com/grafana/mimir/pkg/storegateway"
+	"github.com/grafana/mimir/pkg/util/globalerror"
+)
+
+const (
+	blockRangePeriod = 5 * time.Second
```

Comment:
[nit] Final nit, as promised. `blockRangePeriod` is only used in the test funciton. You can move the constant there. Keep in mind you can define `const` inside the function itself. Just no need for a global const, which may be misused from other tests.

Reply:
Hi @pracucci, 
Could you please do another check and then merge this PR if it looks OK to you?
Thank you